src/helpers/CBV/tests/SingleObject.test.tsx

```diff
@@ -0,0 +1,128 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { mount } from 'enzyme';
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from '../../../store';
+import reducer, {
+  Message,
+  reducerName,
+  removeMessagesAction,
+  selectOneMessage,
+  SEND_MESSAGE,
+  sendMessage,
+  SendMessageAction,
+} from '../../../store/tests/ducks/messages';
+import { SingleObject } from '../SingleObject';
+
+/** register reducers */
+reducerRegistry.register(reducerName, reducer);
+
+interface TestProps {
+  actionCreator?: typeof sendMessage;
+  callbackFunc?: () => void;
+  messages?: Message[];
+}
+
+const TestComponent = (_: TestProps) => <div>mosh</div>;
```

Comment:
Should the `TestComponent` render something from the props? Does that increase test coverage or is that kind of redundant after testing the props alone?

Reply:
It will not improve the test coverage (its already 100%) but I think it is useful for clarity.  I'll add this.